homeassistant/components/starlink/binary_sensor.py

```diff
@@ -0,0 +1,62 @@
+"""Contains sensors exposed by the Starlink integration."""
+
+from __future__ import annotations
+
+from homeassistant.components.binary_sensor import BinarySensorDeviceClass
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import StarlinkUpdateCoordinator
+from .entity import StarlinkBinarySensorEntity, StarlinkBinarySensorEntityDescription
```

Comment:
`StarlinkBinarySensorEntity, StarlinkBinarySensorEntityDescription` should live in `binary_sensor.py` since they are platform specific.

`StarlinkSensorEntity` etc should live in `sensor.py` since it is also platform specific

Reply:
Makes sense, I'll move them while I'm here